package.json

```diff
@@ -142,7 +142,7 @@
     "ngx-base": "2.3.1",
     "ngx-bootstrap": "1.9.3",
     "ngx-fabric8-wit": "6.20.3",
-    "ngx-forge": "0.2.24",
+    "ngx-forge": "^0.2.28",
```

Comment:
You should not have added the carat auto-range. This is a the production application and ranges are locked.

Reply:
@joshuawilson have raised a PR with change https://github.com/fabric8-ui/fabric8-ui/pull/3006